### PR TITLE
fix: connect the visible and invisible areas

### PR DESCRIPTION
**Pull request by daybrush**
## Issue

#451

### CODE REVIEW

File: test/unit/MasonryInfiniteGrid.spec.ts [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+      const outlineLength1 = ig!.getVisibleGroups()[0].grid.getOutlines().start.length;
+      // When
+      // [1, 2]
+      igScrollContainer.scrollTop = 1000;
+
+      await waitEvent(ig!, "renderComplete");
+
+      ig!.getVisibleItems().forEach((item) => {
+        item.element!.style.cssText = "300px";
```
Comment (WoodNeck) [inline test/unit/MasonryInfiniteGrid.spec.ts:54]: I think you should set the more proper value for this, this looks like it's equal to an empty string.

**Review by WoodNeck — APPROVED** [review]
LGTM

**Review by mixed — APPROVED** [review]
LGTM

**Review by ahn3407 — APPROVED** [review]
LGTM

**Pull request merged**